static/css/cf-enhancements.less

```diff
@@ -500,3 +500,18 @@ tbody {
     name: EOF
     eof: true
 */
+
+/**
+ * This is a temporary hack to fix the contact expandables. CF Expandables could use a
+ * larger rework, especially removing duplicated blocks (open/close states, double search
+ * filter buttons based on device size, etc)
+ */
+.expandable__padded .expandable_header {
+  padding-right: 90px;
```

Comment:
Seems a little generous. The links appear to hover around 50px in width. I don't think we'd want to give more than 30px between them and the heading.